plugins/kafka/kafka.cpp

```diff
@@ -54,24 +58,80 @@ namespace KafkaPlugin
     // Static Methods (internal)
     //--------------------------------------------------------------------------
 
+#ifdef _CONTAINERIZED
+    static void applyConfigProps(IPropertyTree * props, const char* configName, RdKafka::Conf* configPtr)
```

Comment:
trivial: perhaps make 'props' const for clarity

Reply:
Good idea; done.